Changes made for cleaner design

MiachofTD opened the pull request:
Moved download text from register.twig to login.twig and changed it to black text via new app-login.css file.  Also created new layout-login.twig to call to new css file.  Adjusted the app.css just for register.twig.

julisana committed:
Merge pull request #5 from MiachofTD/2-login

2 login

MiachofTD committed:
Merge remote-tracking branch 'refs/remotes/origin/2-login'

MiachofTD committed:
Changes made for cleaner design

Moved download text from register.twig to login.twig and changed it to
black text via new app-login.css file.  Also created new
layout-login.twig to call to new css file.  Adjusted the app.css just
for register.twig.

MiachofTD committed:
Merge branch 'master' of github.com:MiachofTD/ACE-web-accounts

julisana commented on public/css/app-login.css:
```diff
@@ -0,0 +1,44 @@
+}
+
+.top-buffer-auth {
+    margin-top: 180px;
```
This one change doesn't warrant making almost completely identical CSS and layout template files. `top-buffer-auth` didn't used to exist before I started making the dashboard page, it was broken into two css classes because the pages had different display requirements. We can do something similar to make the login page have a different top margin than the registration page.

julisana commented on resources/views/layout-login.twig:
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```
See comment on app-login.css. We don't actually need this file. Even if we do decide to go with a separate CSS file for one or more pages, that file can easily be incorporated into `layout.twig` with if statements, rather than duplicating the main layout file.

What makes more sense though, is if the login page needs additional css, we can add css directly to the page (or call additional files) in the header styles block defined in the layout file. That block just needs to be added to the login template.

```twig
{% block headerStyles %}
    We can call any additional CSS files or add in custom CSS here on the login template itself.
{%endblock%}
```

julisana commented on resources/views/auth/login.twig:
```diff
@@ -1,4 +1,4 @@
-{% extends 'layout' %}
+{% extends 'layout-login' %}
```
See comment on layout-login.twig.

MiachofTD committed:
Made correction per PR comments

Removed layout-login.twig and app-login.css.  Added the required CSS to
app.css file.  Adjusted the register.twig and login.twig to call to just
the one CSS file.
